### DISCUSSION TITLE
Group hover: Safari/Chrome IOS doesn't close

### DISCUSSION THREAD

**Discussion by rvmourik** [tailwindlabs/tailwindcss#1842] [Help]
Hi,

In one of our projects we are using a button group hover to show an action menu, however when looking at the page in safari/chrome on an IOS device the dropdown shows when clicking (a.k.a. hovering) the button, but it does not close when clicking somewhere else on the screen.

Is there something that can be done to fix this behaviour.

Thanks in advance

**Comment by anl331** [comment]
@rvmourik did you ever find a solution? I'm stuck trying to figure it out myself.

**Reply by rvmourik** [reply]
To be honest, I never looked at it anymore. So if you still experiencing it, then the issue is still present I guess.

**Reply by pmcp** [reply]
I'm having the same issue with an underline (bottom border) that appears on group-hover. Works as expected on desktop, but on mobile the border appears on click, and doesn't go away anymore (tailwind 2.x)
